airflow/config_templates/default_airflow.cfg

```diff
@@ -675,6 +675,7 @@ git_ssh_known_hosts_configmap_name =
 git_sync_container_repository = k8s.gcr.io/git-sync
 git_sync_container_tag = v3.1.1
 git_sync_init_container_name = git-sync-clone
+git_sync_run_as_user =
```

Comment:
This should stay as the default of 65533 - I don't think it is optional. By specifying a default here we don't need to handle the case when it's not set which makes this change easier.

Reply:
Hi, you have reason, i just change it back to 65533